authors/apps/article_rating/urls.py

```diff
@@ -0,0 +1,13 @@
+from django.urls import path
+from authors.apps.article_rating.views import AverageArticleRatingViewSet, ArticleRatingViewSet
+
+urlpatterns = [
+    path(
+        "<str:slug>/article-rating/", 
```

Comment:
Great work @llwasampijja ,however, I believe it's more efficient to have the rating value also passed in the url instead of having it in the request body. Something close to `<str:slug>/article-rating/<int:rating_value>`, so that we do not burden with organising a json request body.

Reply:
@AtamaZack Let me try to look into it and see if I can be able to implement it like that and see If I will still be able to take full advantage of the ModelViewSets provided by Django Restframework

Reply:
That's a great idea there @AtamaZack , @llwasampijja also remember to update the branch